tests/scalers/graphite-deployment.yaml

```diff
@@ -0,0 +1,359 @@
+---
```

Comment:
Could you move this to `tests/scalers/graphite.test.ts` please, so we are aligned with other scalers.

Reply:
This is actually consistent with how Prometheus does it.

Reply:
OKay, fair enough :)